community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/Configuration.java

```diff
@@ -43,7 +43,7 @@
      * database directory of the imported database, i.e. <into>/bad.log.
      */
     String BAD_FILE_NAME = "bad.log";
-    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 240 );
+    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 480 );
```

Comment:
But we do read from store in quite a lot of our `Stages`, every time we use `ReadRecordsStep`.Would it not be beneficial to not have those reads result in I/O?